src/net/dnsconfig_windows.go

```diff
@@ -30,6 +30,10 @@ func dnsReadConfig(ignoredFilename string) (conf *dnsConfig) {
 	// In practice, however, it mostly works.
 	for _, aa := range aas {
 		for dns := aa.FirstDnsServerAddress; dns != nil; dns = dns.Next {
+			// Only take interfaces whose OperStatus is IfOperStatusUp(0x01) into DNS configs.
+			if aa.OperStatus != 0x01 {
```

Comment:
```suggestion
			if aa.OperStatus != windows.IfOperStatusUp {
```
You'll need to import `syscall/windows` as well.